src/main/java/use_cases/default_game/CustomAssetSetter.java

```diff
@@ -84,20 +85,32 @@ public void setAssets() {
         while (col < MAX_PANEL_COL && row < MAX_PANEL_ROW) {
             int assetNum = mazeAssetNum[col][row];  // go through each element in matrix
             switch (assetNum) {
+                case 0:
+                    // empty tile
+                    break;
                 case 1:
+                    // player?
```

Comment:
If we were to set the player position here it would be 
```suggestion
                   Player.setPlayerX(row);
                   Player.setPlayerY(col);
                   break;
```
since set player is a static method. Another option would be to make the player position same for any difficulty (for example (0,0) and set the player position somewhere else), but I'm okay with either.

Reply:
I really think it's best if we create a hashmap for the cases, as it would make the custom asset setter more efficient and general.